### ISSUE TITLE
All commands except cd and ls hangs

### ISSUE THREAD

**Issue by strowk** [kubernetes-sigs/kui#3371]
**Describe the bug**
I only recently installed kui at Windows 10 and I noticed that almost all commands hang indefinitely. So far I managed only to find `ls` and `cd`, which do not hang. `ls` gives error that it is not recognized as an internal or external command (looks like a regular response from `cmd`, even though I have git bash installed, kui for some reason uses cmd).

**Steps to reproduce the behavior**

1. Download Kui
2. Unpack `Kui-win32-x64` and run Kui.exe
3. run `cd ..` -> success
4. run ls -> immediate error
5. run `kubectl version` -> stuck indefinitely
6. close window, open Kui.exe again
7. try to enter something completely random -> stuck indefinitely

**Expected behavior**
Command should not hang.

**Screenshots**
![image](https://user-images.githubusercontent.com/25692644/70826453-40cc4080-1de7-11ea-9f06-ed4527a7d1ef.png)

**System**

- Kui version: 6.1.5
- Operating system: Windows 10

**Comment by starpit** [comment]
i believe we had a buggy set of builds (related: #3374 ). we just triggred a rebuild. the new builds seem better. i will test the latest build more thoroughly on windows, in a few hours. so far, though, linux and mac seem to have restored function with the latest builds

**Comment by starpit** [comment]
i just confirmed via manual testing on a windows 10 test VM. the latest downloads seem to have resolved the buggy build.

**Comment by strowk** [comment]
It's still reproducing on latest downloads. Error in logs:
```
could not exec Error: File not found: 
    at new WindowsPtyAgent (C:\progs\Kui-win32-x64\resources\app\node_modules\node-pty-prebuilt-multiarch\lib\windowsPtyAgent.js:77)
    at new WindowsTerminal (C:\progs\Kui-win32-x64\resources\app\node_modules\node-pty-prebuilt-multiarch\lib\windowsTerminal.js:49)
    at spawn (C:\progs\Kui-win32-x64\resources\app\node_modules\node-pty-prebuilt-multiarch\lib\index.js:28)
    at 50.4e76fadca9e54e14b002.bundle.js:1
    at Generator.next (<anonymous>)
    at r (50.4e76fadca9e54e14b002.bundle.js:1)
```

**Comment by starpit** [comment]
thanks for digging in on this! it looks like we have an issue when Kui is launched from cmd or powershell; it seems to be ok when launched from git-bash.

we have a fix in master now #3444 #3442 that should improve powershell support. even though it's merged, i will keep this issue open until we release a new build. thanks again!

<img width="316" alt="Screen Shot 2020-01-03 at 11 46 50 AM" src="https://user-images.githubusercontent.com/4741620/71740772-1e987200-2e2b-11ea-9320-c920adbfc5aa.png">

**Comment by strowk** [comment]
I actually did not 'launched from cmd or powershell', I just double-clicked on Kui.exe.. Is it not the right way to run it? :) 
Docs mention example command 'kubectl kui get pods --ui', but that (predictably) gives
```
Error: unknown command "kui" for "kubectl"
```
Maybe some additional actions should be described in documentation regarding what should be done with downloaded archive? Right now it's a bit obscure...
